### DISCUSSION TITLE
NextJS: 13 How long does "force-cach" option cache?

### DISCUSSION THREAD

**Discussion by KeisukeNagakawa** [vercel/next.js#42559] [Help]
When using `fetch`, default option is "force-cache" and I would like to know how long it caches. Thanks!

**Comment by icyJoseph** [comment]
I may be wrong here but I think these options pretty much follow this list: https://developer.mozilla.org/en-US/docs/Web/API/Request/cache

> force-cache — The browser looks for a matching request in its HTTP cache.
> If there is a match, fresh or stale, it will be returned from the cache.
> If there is no match, the browser will make a normal request, and will update the cache with the downloaded resource.

Given that this is documentation for the browser fetch, we could feel a bit doubtful, but for example, these are passed to `undici`, which conforms to the Fetch spec AFAIK. 

In practical terms, once it fetches, it never fetches again, for the lifetime of the node process I'd guess.
